### PR TITLE
Cultural show page

### PR DESCRIPTION
**Pull request by summer-cook**
# Summary
This PR edits the theming on the cultural show page. It moves the relationships so that they are in their own column. It also makes it so the relationships box does not show up when no relationships are present

## Related
#374 

# Screenshots / Video

<details>

When a work has no relationships:
no-relationships

When a work has relationships/user is logged in:
relationships-logged-in

When a work has the UV:
screencapture-tenant-pals-docker-concern-generic-works-6a61d019-d596-4777-a7ee-f4dd5a35a68c-2021-09-21-13_53_41

Video (shows the view when a user is not logged in and relationships exist, also demonstrates responsiveness at smaller screen sizes): https://share.getcloudapp.com/12uvmKbe

</details>

# Acceptance Criteria
- [ ] Make wireframes/do design work re: cultural show page
- [ ] No 3 column layout when UV is not showing
- [ ] Relationships box should not show when the end user cannot see any of the relationships
- [ ] Relationships box should have label and values on separate lines / stacked so the text doesn't break at odd places
- [ ] Responsive on all screen sizes

**_Code Contribution Courtesy of Palni-Palci_**

@samvera/hyku-code-reviewers

### CODE REVIEW

File: app/views/themes/cultural_show/hyrax/base/show.html.erb [hunk 1]
```diff
@@ -12,26 +12,29 @@
...
-          <div class="<%= @presenter.universal_viewer? ? 'col-sm-8' : 'col-sm-6' %>">
+          <div class="<%= @presenter.universal_viewer? ? 'col-sm-6' : 'col-sm-6' %>">
             <%= render 'work_description', presenter: @presenter %>
             <%= render 'metadata', presenter: @presenter %>
           </div>
-          <div class="<%= @presenter.universal_viewer? ? 'col-sm-4' : 'col-sm-3' %>">
+          <div class="col-xs-12 relationships-container">
             <div class='relationships-box'>
               <%= render 'relationships', presenter: @presenter %>
+              <% if @presenter.class == Hyrax::OerPresenter %>
```
Comment (labradford) [inline app/views/themes/cultural_show/hyrax/base/show.html.erb:32]: OER is a custom worktype for PALNI PALCI code base, so this should be removed (lines 32-34)

**Review by labradford — CHANGES_REQUESTED** [review]
Remove OER reference and fix the css linting and this should be ready for merge

**Comment by summer-cook** [comment]
alright @labradford , got this one to pass. not sure why it failed in the first place bc i just had to change it and then change it back again to what it was

File: app/assets/stylesheets/hyku.scss [hunk 2]
```diff
@@ -19,29 +19,28 @@
...
-
   .home-tabs-left {
     min-width: 50%;
   }
 }
 
 #banner.jumbotron {
-  background: image_url('white-cloud-background.jpg') no-repeat;
-  background-color: $gray-lighter;
+  background: image_url("Hyku_Commons_Logo.png") no-repeat;
```
Comment (labradford) [inline app/assets/stylesheets/hyku.scss:38]: This should be the white-cloud-background file and not the hyku commons logo

**Review by labradford — CHANGES_REQUESTED** [review]
can you change the background image back to the default hyku one and then this should be ready

File: app/assets/stylesheets/hyku.scss (hunk 2 above)
Comment (summer-cook) [reply app/assets/stylesheets/hyku.scss]: done 👍

**Review by labradford — APPROVED** [review]

**Pull request merged**